Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -1202,8 +1314,14 @@ Tileset::TraversalDetails Tileset::_visitTile(
 
   // At least one descendant tile was added to the render list.
   // The traversalDetails tell us what happened while visiting the children.
-  if (!traversalDetails.allAreRenderable &&
-      !traversalDetails.anyWereRenderedLastFrame) {
+  if ((!traversalDetails.allAreRenderable &&
+       !traversalDetails.anyWereRenderedLastFrame) ||
+      (_options.enableLodTransitionPeriod &&
+       _options.kickDescendantsWhileFadingIn &&
+       lastFrameSelectionResult == TileSelectionState::Result::Rendered &&
+       tile.getContent().isRenderContent() &&
+       tile.getContent().getRenderContent()->getLodTransitionFadePercentage() <
+           1.0f)) {
```

Comment:
This part of the code is worth reviewing in particular: the changes to the kick logic. This allows us to keep descendants off the render list if the tile is still fading in. The descendants may still continue to load though. If this option is disabled, the fading-in tile will pop in at full opacity so that the rendered descendant can start fading in.